resume.html

```diff
@@ -0,0 +1,97 @@
+				<div>2015 summer, fall: original graphics printing: subcontractor; 5+ brands: website, signage, decals, tshirts</div>
+			</section>
+			<section>
+				<header>events</header>
+				<p>in lieu of more experience, heres a list of events I have attended</p>
+				<p>member of: detroit.js, sem.js, detroit react developers, jamstack nyc, new york web performance group</p>
+				<p>2017/04/06 - theNewDynamic @ Carrot Creative, NY,NY<br>cms as code, pushing the limits w/ static site tools</p>
+				<p>2017/04/27 - Active Ingredients @ Google, SF,CA<br>activeingredients.info by Netlify</p>
+				<p>2017/10/02 - JAMstack NYC @ O’Reilly’s Velocity, NY,NY<br>Git-based or API-driven CMS - by Chris Macrae of forestry.io</p>
+				<p>2017/10/11 - Elm Philadelphia @ Blackfynn, Philly,PA<br>Beginners Night! - by Richard Feldman</p>
+				<p>2017/11/06 - Detroit Javascript @ Detroit Labs, DET,MI<br>Lets Kill CSS: Pros and Cons of CSS in JS - by Steve Barman and Dennay Bedard</p>
+				<p>2018/11/13 - SE Michigan JavaScript @ Barracuda Networks, Ann Arbor,MI<br>Exploring Vue.js, an alternative to Angular and React - by Jacob Bolton</p>
+				<p>2018/01/31 - Detroit Javascript @ Detroit Labs, DET,MI<br>Mob Coding! Build a server-side NextJS blog</p>
+				<p>2018/02/22 - Detroit Javascript @ Detroit Labs, DET,MI<br>Programming with Attention Deficit Disorder - by Steve Barman</p>
+				<p>2018/03/21 - Detroit Javascript @ OtterBase, Livonia,MI<br>Group coding with exercism.io - by Mike Matola</p>
+				<p>2018/03/27 - HackerNest Detroit @ i3Detroit, Ferndale,MI<br>March Social</p>
+				<p>2018/04/09 - SE Michigan JavaScript @ Barrcuda Networks, Ann Arbor,MI<br>ASTronomically Improving your Developer Experience: Abstract Syntax Tree Primer - by Bernardo Pacheco of Dominos Pizza</p>
+				<p>2018/04/24 - New York Web Performance Group @ Spotify, NY,NY<br>Building a Static HTML & Web App Frankenstein - by Jeff Escalante of Hashicorp</p>
+				<p>2018/05/01 - Detroit Javascript @ RouteOne, Farmington Hills,MI<br>Securing VueJS Applications with JSOn Web Tokens - by Ado Kukic of Auth0</p>
+				<p>2018/06/18 - Detroit Javascript @ Detroit Labs, DET,MI<br>a FUNctional Javascript Makeover - by Mike Schutte</p>
+				<p>2018/07/10 - New York Web Performance Group @ Priceline, NY,NY<br>Meet for SPEED: Lets talk Webpack - by Sergey Chernyshev of Meetup.com and Malek Hakim of Priceline.com</p>
+				<p>2018/07/18 - New York Web Performance Group @ Squarespace, NY,NY<br>FaCSSt: CSS & Performance - by Harry Roberts, csswizardy.com</p>
+			</section>
+			<section>
+				<header>skills</header>
+				<p>
+					hypertext markup language, cascading style sheets, javascript<br>
+					html5, liquid, yaml, toml, markdown, haml, jade, pug, json, reshape<br>
+					css3, sass, less, stylus, postcss, css in js, styled components<br>
+					js, jquery, node, lodash, babel, es6, react, vue, elm, ramda, reasonml<br>
```

Comment:
Typically using css to bound the width is more idiomatic than manually adding line breaks

Reply:
i switched it to divs